ConfigurationSystem/Service/ConfigurationHandler.py

```diff
@@ -23,7 +27,7 @@ def export_getCompressedData( self ):
     sData = gServiceInterface.getCompressedConfigurationData()
     return S_OK( sData )
 
-  types_getCompressedDataIfNewer = [ types.StringTypes ]
+  types_getCompressedDataIfNewer = [ StringTypes ]
```

Comment:
I also think you are correct. In python 3 they removed from the "types" module everything that can be accessed via builtins, and this includes types.StringType(s) or ListType and so on. 

We may want to change all of these everywhere. And BTW this is something that would need to be done in the prospect of moving to python 3, as the 2to3 tool does not do a good job with it:

```
 toffo@pclhcb181  ~/aaaRumenta  more pippo.py
import types

type('a') == types.StringType

isinstance('a', basestring)
```

```
toffo@pclhcb181  ~/aaaRumenta  2to3 pippo.py

@@ -1,5 +1,5 @@
 import types
 
-type('a') == types.StringType
+type('a') == bytes
 
-isinstance('a', basestring)
+isinstance('a', str)
```

(note: ``basestring`` in python 3 is simply ``str``)